drivers/SmartThings/zwave-sensor/src/aeotec-multisensor/multisensor-6/init.lua

```diff
@@ -34,7 +34,8 @@ local function wakeup_notification(driver, device, cmd)
     device:set_field("__wakeup_interval_get_sent", true)
   end
   device:send(Configuration:Get({parameter_number = PREFERENCE_NUM}))
-  device:refresh()
+  local version = require "version"
+  if version.api ~= 6 then device:refresh() end
```

Comment:
I ended up doing it this way along the way, but I could instead exclude the multisensor 6 and 7 in the `apiv6_bugfix` can_handle function like I did for a few other devices that have their own wakeup_notification handler. Looking at it this morning it seems better to do this to keep the bugfix local to its own subdriver, so I will go forward with that.